version: 1

#!RESULT ERROR
#!CONNECTION ldap://eap.sysnet.cz:389
#!DATE 2020-03-31T11:09:18.267
#!ERROR ENTRY_ALREADY_EXISTS: failed for MessageType : ADD_REQUEST Message ID : 28     Add Request : Entry     dn: dc=cz     objectClass: extensibleObject     objectClass: domain     objectClass: top     dc: cz : ERR_250_ENTRY_ALREADY_EXISTS dc=cz already exists!
dn: dc=cz
objectClass: extensibleObject
objectClass: domain
objectClass: top
dc: cz

